easybuild/easyblocks/d/dolfin.py

```diff
@@ -47,24 +47,25 @@
 class EB_DOLFIN(CMakePythonPackage):
     """Support for building and installing DOLFIN."""
 
+    @staticmethod
+    def extra_options():
+        extra_vars = CMakePythonPackage.extra_options()
+        extra_vars['separate_build_dir'][0] = True
+        extra_vars['build_type'][0] = 'Release'
```

Comment:
Same here, if toolchainopts has debug True, should probably be Debug

Reply:
I'd actually solve this in the cmakemake block: when it is none (default) then use either release or debug depending on the toolchainopt 
This would be a change to current behavior where it is not currently set, but imo is the right thing as it is the equivalent of what is done for other build systems. See #911 for discussion on defaulting to release

Reply:
Yes, doing this in cmakemake is clearly the "right" thing to do.

Note though that for dolfin you changed it from the previous Debug to Release
Although that one is kind of wrong anyway...

Reply:
If you are ok I'd change that in the cmakemake eb so release will become the default effectively. I'd prefer that very much

In this case this is correct as the debug was unintended: https://github.com/easybuilders/easybuild-easyblocks/issues/1920